### PR TITLE
Updates the frontend file

### PR DESCRIPTION
**Pull request by ankur0904**
I made some changes in the frontend files of this project. This is a webpage that has the potential to manage to vote.
One of them is I added the border-radius to both the images due to the images exactly fit with the background image.
And next things are about the buttons I made some changes in the CSS part of the button, e.g. I added padding to the button to increase its visibility etc.
I managed all the positioning of all the div so that all div exactly fit with their place.

This is all about my contribution.

### CODE REVIEW

**Comment by ankur0904** [comment]
Screenshot from 2022-10-01 21-47-53

**Comment by Vikash-8090-Yadav** [comment]
Give a proper decription about the change in  this pr as  this gonna see by the  maintainer of the hacktoberfest.
And on the basis of this your pr will accept  or reject . 
and my repo also depends on yours description . 
Hope u understand .

**Comment by ankur0904** [comment]
I added the description.

File: Level0/Voting/index.html [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
+			Total vote for candidate 1
 		</button>
-		<button id = "show-vote2">
-			TOTAL VOTE FOR CANDIDATE 2
+		<button id="show-vote2">
+			Total vote for candidate 2
 		</button>
 	</div>
 </body>
+
```
Comment (Vikash-8090-Yadav) [inline Level0/Voting/index.html]: Do  one thing  increase    the font size of the  TOTAL VOTE FOR CANDIDATE 1 and TOTAL VOTE FOR CANDIDATE 2.
 and also check if this responsive or not ,and if not please fix this .
@ankur0904 Hope u understand .

**Comment by Vikash-8090-Yadav** [comment]
please close this one..

**Pull request closed without merging**